hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestEditLogTailer.java

```diff
@@ -429,19 +432,43 @@ public void testStandbyTriggersLogRollsWhenTailInProgressEdits()
       waitForStandbyToCatchUpWithInProgressEdits(standby, activeTxId,
           standbyCatchupWaitTime);
 
+      long curTime = standby.getNamesystem().getEditLogTailer().getTimer()
+          .monotonicNow();
+      // Set TestTimer that returns (curTime + 1 sec) ms while calling
+      // timer.monotonicNow(). This returned value is used by
+      // EditLogTailer.tooLongSinceLastLoad() to determine whether
+      // EditLogTailer should roll the log. Since this test is time
+      // and resource sensitive, we are using custom Timer instance
+      // so that tooLongSinceLastLoad() always returns false
+      // until we change Timer instance in EditLogTailer.
```

Comment:
Can we simplify this comment to just:
```
// move time forward by 1 second, which is not enough to trigger a log roll
```
Better yet, we could save the log roll time in line 405 above:
```
long logRollPeriod = standbyCatchupWaitTime + noLogRollWaitTime + 1;
```
and then set the time to `curTime + TimeUnit.SECONDS.toMillis(logRollPeriod / 2)` -- I think this should be sufficiently self-explanatory to not require a comment, since you can clearly see we're only moving time by half the log-roll period.